agent/command/git_test.go

```diff
@@ -720,13 +741,15 @@ func (s *GitGetProjectSuite) TestBuildModuleCommand() {
 	cmds, err = c.buildModuleCloneCommand(conf, opts, "main", module)
 	s.NoError(err)
 	s.Require().Len(cmds, 7)
-	s.Equal("set -o xtrace", cmds[0])
-	s.Equal("set -o errexit", cmds[1])
-	s.Equal("git clone 'git@github.com:evergreen-ci/sample.git' 'module'", cmds[2])
-	s.Equal("cd module", cmds[3])
-	s.Regexp("^git fetch origin \"pull/1234/merge:evg-merge-test-", cmds[4])
-	s.Regexp("^git checkout 'evg-merge-test-", cmds[5])
-	s.Equal("git reset --hard 1234abcd", cmds[6])
+	s.True(utility.StringSliceContainsOrderedPrefixSubset([]string{
+		"set -o xtrace",
+		"set -o errexit",
+		"git clone 'git@github.com:evergreen-ci/sample.git' 'module'",
+		"cd module",
+		"^git fetch origin \"pull/1234/merge:evg-merge-test-",
```

Comment:
Since the `^` is a regexp pattern, does this work with `utility.StringSliceContainsOrderedPrefixSubset`? Seems like we want to get rid of the leading `^`, since it's already matching a prefix, no?

Reply:
Yeah, I meant to remove those and I used the prefix version to emulate that, I'll remove them